.github/workflows/main.yaml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-blockdiag:
+    runs-on: ubuntu-18.04
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    - name: Install python 3.8.1
+      uses: gabrielfalcao/pyenv-action@v5
+      with:
+        default: 3.8.1
+        command: pip install -U pip  # upgrade pip after installing python
+
+    - name: Create virtualenv for python 3.8.1
+      run: pyenv local 3.8.1
+
+    - name: Install BlockDiag
+      run: make installBlockDiag
+
+    - name: Test BlockDiag
+      run: make testBlockDiag
+
+  test-containers:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up JDK 12
+      uses: actions/setup-java@v1
+      with:
+        java-version: 12
```

Comment:
I think we should use the Java 11 LTS. Currently we are using Java 8 but not for long: https://github.com/yuzutech/kroki/pull/155